desktop/core/src/desktop/js/utils/useDebounce.test.ts

```diff
@@ -0,0 +1,100 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import { renderHook, act } from '@testing-library/react';
+import useDebounce from './useDebounce';
+
+type SomeFunction = (...args: string[]) => void;
+
+const mockFunction: jest.Mock<SomeFunction> = jest.fn();
+let delay: number;
```

Comment:
Can we change this to `const delay = 1000` as beforeEach is setting this variable each time to `1000` which is equivalent to suggested statement.